.github/workflows/unit_tests.yaml

```diff
@@ -103,7 +103,7 @@ jobs:
           concretizer: original
         - python-version: 3.6
           concretizer: original
-        - python-version: '3.10'
+        - python-version: 3.9
```

Comment:
What was wrong with this? Also, we should make the rest of these strings.

Reply:
I didn't want to rerun CI on each of the 600 PRs still pending because we changed the required Python on the deprecated concretizer.